### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by JaSei**
* fix & improve documentation
* configuration changes:
    * get_from_hcp renamed to s3_enabled
    * hcp_credentials renamed to s3_credentials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +67,21 @@ Stor is an HTTP API to primary storage. You provide a SHA256 hash and get the fi
...
+#### 200 OK
+
+File exists
+
+Headers:
+
+    Content-Length - file size of file
+    Last-Modified - last modification time
+
+GET return content of file in body
```
Comment (jendakol) [inline README.md:79]: I don't understand.

**Review by jendakol — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,6 +15,48 @@ Stor - Save/retrieve a file to/from primary storage
 
 Stor is an HTTP API to primary storage. You provide a SHA256 hash and get the file contents, or you provide a SHA256 hash and a file contents and it gets stored to primary storages.
 
+## How to use?
+
+### docker way
+
+    docker run -p $PWD/config.json.example:/etc/stor.conf -e CONFIG_FILE=/etc/stor.conf avastsoftware/stor:TAG
```
Comment (jendakol) [inline README.md:22]: `-p`? Is that correct?

Comment (petr-k) [reply README.md:22]: Should be `-v`

File: README.md (hunk 1 above)
Comment (JaSei) [reply README.md:79]: ups copy-paste bug - thanks

File: README.md (hunk 2 above)
Comment (JaSei) [reply README.md:22]: good point

**Pull request merged**